serpentTools/tests/test_container.py

```diff
@@ -0,0 +1,86 @@
+"""Test the container object. """
+
+import unittest
+from serpentTools.objects import containers
+from serpentTools.parsers import DepletionReader
+
+
+class containerTester(unittest.TestCase):
+    """ Class to test the container object """
+
+    @classmethod
+    def setUpClass(cls):
+        container = DepletionReader(None)
```

Comment:
Pedantic/non-crucial request: Can you just pass `DepletionReader(None)` as the first argument to `containers.HomogUniv`? There isn't a need to store the Depletion reader after the creation of the `HomogUniv`

Reply:
Done